manubot/cite/citekey.py

```diff
@@ -249,13 +249,16 @@ def shorten_citekey(standard_citekey: str) -> str:
 
 
 def citekey_to_csl_item(
-    citekey, prune=True, manual_refs={}, log_level: tp.Union[str, int] = "WARNING"
+    citekey, prune=True, manual_refs=None, log_level: tp.Union[str, int] = "WARNING"
```

Comment:
This is a good change. I recently was reminded of the unexpected behavior of this pattern.